--- matio-1.3.4/src/Makefile.am.orig	2008-05-18 23:45:41.000000000 +0200
+++ matio-1.3.4/src/Makefile.am	2012-01-29 19:31:12.697914151 +0100
@@ -15,6 +15,7 @@
 noinst_HEADERS      = matio_private.h mat4.h mat5.h
 include_HEADERS     = matio.h matioConfig.h
 lib_LTLIBRARIES     = libmatio.la
+libmatio_la_LIBADD  = $(ZLIB_LIBS)
 libmatio_la_SOURCES = snprintf.c endian.c io.c $(ZLIB_SRC) read_data.c \
                       mat5.c mat4.c mat.c
 
